app/views/modals/_renew_session.html.erb

```diff
@@ -0,0 +1,26 @@
+<a href="#renew-modal" style="display:none;" id="renew-modal-open-button" class="usa-button" aria-controls="renew-modal" data-open-modal></a>
+
+<div
```

Comment:
did you try this using the [<dialog /> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) it does nice stuff like input trapping and you can style (animate) the show/hide  transition too

Reply:
I was just basing it off USWDS components. I didn't see dialog mentioned in the docs unless I missed it